Use bower's sinon.js release directly from URL,

tomalec opened the pull request:
as the bower repo is outdated.

This should make updates easier. However, it still uses the version as before.

I'd love to bump the Sinon to the latest version (1.17.1-> 4.1.1), but I separated that to reduce the risk of migration, and increase the chance for merging.

<!--
  Thanks for the PR!

  If this change has a user visible change (including
  bug fixes, new features, etc) please describe the change in
  CHANGELOG.md.

  If the change is an entirely package-internal reshuffling/refactoring
  should the change not be described in the CHANGELOG.

  Consider also updating the README.

  More info: http://keepachangelog.com/en/0.3.0/
 -->

 - [ ] CHANGELOG.md has been updated

tomalec committed:
Use bower's sinon.js release directly from URL,

as the bower repo is outdated.

This should make updates easier. However, it still uses the version as before.

mroderick commented on bower.json:
```diff
@@ -32,7 +32,7 @@
     "mocha": "^3.1.2",
     "sinon-chai": "^2.7.0",
-    "sinonjs": "^1.14.1",
+    "sinonjs": "http://sinonjs.org/releases/sinon-1.17.1.js",
```
We (the Sinon maintainers) would much prefer if you would use the releases from the CDN, than from the documentation site (which is graciously hosted by GitHub).

```
https://cdn.jsdelivr.net/npm/sinon@1.17.1/pkg/sinon.js
```

tomalec committed:
Use CDN URL for sinon.js package

As suggested by @mroderick 
https://github.com/Polymer/web-component-tester/pull/650/files#r155719863

justinfagnani commented:
@usergenic I think you most recently dealt with Sinon versions, does this look right to you?

tomalec commented:
@usergenic , @justinfagnani any chance to get it there?

aomarks commented:
Closing this pull request because this repo is now archived. If this PR is still active, please submit a new one at https://github.com/Polymer/tools.
